homeassistant/components/sfr_box/quality_scale.yaml

```diff
@@ -0,0 +1,96 @@
+rules:
```

Comment:
```py
class SFRDataUpdateCoordinator[_DataT](DataUpdateCoordinator[_DataT | None]):
    """Coordinator to manage data updates."""

    config_entry: SFRConfigEntry

    def __init__(
        self,
        hass: HomeAssistant,
        config_entry: SFRConfigEntry,
        box: SFRBox,
        name: str,
        method: Callable[[SFRBox], Coroutine[Any, Any, _DataT | None]],
```
I don't think you need to add `| None` here, as `_DataT` already would be the return type of the method passed in, and that one already has `| None`